duet/explores/desktop_new_profile.explore.lkml

```diff
@@ -1,5 +1,57 @@
 include: "../views/*.view.lkml"
 
+view: distinct_new_profiles {
+  derived_table: {
+    sql:
+    SELECT
+    ROW_NUMBER() OVER (PARTITION BY client_id, DATE(submission_timestamp)) AS rn,
+    client_id,
+    submission_timestamp
+  FROM
+    `mozdata.telemetry.new_profile`
+  WHERE
+      DATE(submission_timestamp) <= DATE_SUB(
+      IF({% parameter desktop_new_profile.previous_time_period %},
+        -- if the data for the previous time period is requested,
+        -- shift dates by the date range provided via the 'date' filter
+        DATE(DATE_ADD(
+          DATE({% date_end desktop_new_profile.date %}),
+          INTERVAL DATE_DIFF(DATE({% date_start desktop_new_profile.date %}), DATE({% date_end desktop_new_profile.date %}), DAY) DAY)),
+        DATE({% date_end desktop_new_profile.date %})),
+      -- if the most recent week is to be ignored, shift date range by 9 days
+      INTERVAL IF({% parameter desktop_new_profile.ignore_most_recent_week %}, 9, 0) DAY)
+    AND
+    DATE(submission_timestamp) > DATE_SUB(
+      IF({% parameter desktop_new_profile.previous_time_period %},
+        -- if the data for the previous time period is requested,
+        -- shift dates by the date range provided via the 'date' filter
+        DATE(DATE_ADD(
+          DATE({% date_start desktop_new_profile.date %}),
+          INTERVAL DATE_DIFF(DATE({% date_start desktop_new_profile.date %}), DATE({% date_end desktop_new_profile.date %}), DAY) DAY)),
+        DATE({% date_start desktop_new_profile.date %})),
+      -- if the most recent week is to be ignored, shift date range by 9 days
+      INTERVAL IF({% parameter desktop_new_profile.ignore_most_recent_week %}, 9, 0) DAY) AND
+    payload.processes.parent.scalars.startup_profile_selection_reason = 'firstrun-created-default';;
+  }
```

Comment:
Why not get the most recent row number in the view instead of exposing the row number and filtering into the explore? Are there other uses for the row number?

Reply:
The row number is used to ensure we count duplicate entries only once. Looker doesn't support aggregate function in dimension definitions. So having `ROW_NUMBER() OVER (PARTITION BY client_id, DATE(submission_timestamp))` as a dimension in `desktop_new_profile` is not possible. 

It would have been possible to rewrite `desktop_new_profile` as a `derived_table`, however then taking advantage of the automatically generated view via lookml-generator isn't possible anymore.

Having a separate view with distinct clients used to ensure we don't have duplicates seemed the best way forward here.